buildSrc/src/integTest/groovy/org/elasticsearch/gradle/internal/InternalBwcGitPluginFuncTest.groovy

```diff
@@ -19,33 +19,60 @@
 
 package org.elasticsearch.gradle.internal
 
-import org.elasticsearch.gradle.fixtures.AbstractGradleFuncTest
+import org.elasticsearch.gradle.fixtures.AbstractGitAwareGradleFuncTest
 import org.gradle.testkit.runner.TaskOutcome
 
-class InternalBwcGitPluginFuncTest extends AbstractGradleFuncTest {
+class InternalBwcGitPluginFuncTest extends AbstractGitAwareGradleFuncTest {
 
     def setup() {
-        setupLocalGitRepo()
-    }
-
-    def "current repository can be cloned"() {
-        given:
-        internalBuild();
+        internalBuild()
         buildFile << """
             import org.elasticsearch.gradle.Version;
             apply plugin: org.elasticsearch.gradle.internal.InternalBwcGitPlugin  
             
             bwcGitConfig {
-                 bwcVersion = project.provider { Version.fromString("7.10.0") }
-                 bwcBranch = project.provider { "7.x" }
+                 bwcVersion = project.provider { Version.fromString("8.1.0") }
+                 bwcBranch = project.provider { "8.0" }
                  checkoutDir = project.provider{file("build/checkout")}
             }
         """
+    }
+
+    def "current repository can be cloned"() {
         when:
         def result = gradleRunner("createClone", '--stacktrace').build()
         then:
         result.task(":createClone").outcome == TaskOutcome.SUCCESS
-        file("build/checkout/build.gradle").exists()
-        file("build/checkout/settings.gradle").exists()
+        file("cloned/build/checkout/build.gradle").exists()
+        file("cloned/build/checkout/settings.gradle").exists()
+    }
+
+    def "can resolve checkout folder as project artifact"() {
```

Comment:
@jakelandis this functional tests basically shows an example on how to resolve the checkoutDir as artifact. you can declare a dependency in the es build by declaring 

```
dependencies {
    myConfig project(path:':distribution:bwc:minor', configuration:'checkoutDir')
}

```